openpype/modules/sync_server/sync_server_module.py

```diff
@@ -1792,11 +1796,32 @@ def _update_site(self, project_name, representation_id,
 
             Used for refactoring ugly reset_provider_for_file
         """
-        query = {
-            "_id": ObjectId(representation_id)
-        }
+        representation_ids = [ObjectId(representation_id)]
 
-        self.connection.database[project_name].update_one(
+        # Add hero version
+        representation = get_representation_by_id(
+            project_name, representation_id
+        )
+        representation_parents = get_representation_parents(
+            project_name, representation
+        )
+        hero_version = get_hero_version_by_subset_id(
+            project_name, representation_parents[1]["_id"], fields=["_id"]
+        )
+        if hero_version:
+            representations = get_representations(
+                project_name,
+                version_ids=[hero_version["_id"]],
+                fields=["_id"],
+                context_filters={"ext": [representation["context"]["ext"]]},
```

Comment:
I don't think it should be filtrered by extension in context but by `"name"`.

Reply:
I don't think so, I have representations which have name 'thumbnail'. 
(But I have also some old representations without `representation["context"]["ext"]`)

Reply:
So it should be filtered by representation name instead of extension. Extension on context can not be unique (there can be more then one representation with extension `.mov` under single version). So this is unsave and could potentially cause sync of more representations.